docs/index.rst

```diff
@@ -134,6 +134,67 @@ u'Donnerstag, 5. M\xe4rz 1987 17:12'
 
 For more format examples head over to the `babel`_ documentation.
 
+Formatting Numbers
+----------------
+
+To format numbers you can use the :func:`format_number`,
+:func:`format_decimal`, :func:`format_currency`, :func:`format_percent` and :func: `format_scientific`
+functions.
+
+To play with the date formatting from the console, you can use the
+:meth:`~flask.Flask.test_request_context` method:
+
+>>> app.test_request_context().push()
+
+Here some examples:
```

Comment:
> here *are* some examples

Reply:
What I need do here?

Reply:
I copy this phare from this doc.

Needs change anothers places?

In my 'place' I changed.